src/include/uapi/ipc.h

```diff
@@ -329,8 +329,8 @@ struct sof_ipc_dai_dmic_params {
 	uint32_t fifo_fs_a;  /* FIFO A sample rate in Hz (8000..96000) */
 	uint32_t fifo_fs_b;  /* FIFO B sample rate in Hz (8000..96000) */
 	/* TODO: FIFO word lengths can be retrieved from SOF_DAI_FMT */
-	uint16_t fifo_bits_a; /* FIFO A word length (16 or 24) */
-	uint16_t fifo_bits_b; /* FIFO B word length (16 or 24) */
+	uint16_t fifo_bits_a; /* FIFO A word length (16 or 32) */
```

Comment:
can you clarify the TODO just above? I don't get what this means and wonder if we can clean this up while we are at it?

Reply:
Isn't sof_dai_fmt for something else? Current implementation ignores that anyway. Also if FIFOs can have 2 different word lengths how are we going to put it into 1 SOF_DAI_FMT. We have to keep it like now in 1 config or use 1 dai per fifo what I guess we don't want, because it would be misleading.